code/modules/client/verbs/looc.dm

```diff
@@ -44,6 +41,9 @@ AUTH_CLIENT_VERB(looc, msg as text)
 			to_chat(src, span_bolddanger("Advertising other servers is not allowed."))
 			log_admin("[key_name(src)] has attempted to advertise in LOOC: [msg]")
 			return
+		if (HAS_TRAIT(mob, TRAIT_RESTRAINED) || HAS_TRAIT(mob, TRAIT_INCAPACITATED) || HAS_TRAIT(mob, TRAIT_IMMOBILIZED) || HAS_TRAIT(mob, TRAIT_MUTE))
```

Comment:
This should probably be a config check as well so we can enable-disable if it's either unsatisfactory or breaks.

I imagine this PR isn't really the end for future changes to LOOC given the state of it rn.

(Nonblocking)